src/view/uniprotkb/components/__tests__/__snapshots__/CatalyticActivityView.spec.tsx.snap

```diff
@@ -47,9 +47,6 @@ exports[`CatalyticActivityView component should render catalytic activity 1`] =
         <h5>
           Manual assertion based on experiment
         </h5>
-        <div>
```

Comment:
I noticed we're not catching the publications in this component's snapshots - though they are there when looking at the site. It's probably not necessary but we could in theory populate this with a publication snapshot.